<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Q3DWindow</class>
 <widget class="QMainWindow" name="Q3DWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>580</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Live Exporter - Qgis2threejs</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="Q3DView" name="webView" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>757</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuScene">
    <property name="title">
     <string>&amp;Scene</string>
    </property>
    <widget class="QMenu" name="menuCamera">
     <property name="title">
      <string>Camera</string>
     </property>
     <addaction name="actionPerspective"/>
     <addaction name="actionOrthogonal"/>
    </widget>
    <widget class="QMenu" name="menuControls">
     <property name="title">
      <string>Controls</string>
     </property>
     <addaction name="actionOrbit"/>
    </widget>
    <addaction name="actionWorld_Settings"/>
    <addaction name="menuCamera"/>
    <addaction name="menuControls"/>
    <addaction name="separator"/>
    <addaction name="actionReload"/>
    <addaction name="separator"/>
    <addaction name="actionResetCameraPosition"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <widget class="QMenu" name="menuPanels">
     <property name="title">
      <string>Panels</string>
     </property>
    </widget>
    <addaction name="menuPanels"/>
    <addaction name="separator"/>
    <addaction name="actionAlwaysOnTop"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionExportToWeb"/>
     <addaction name="actionSTL_format"/>
    </widget>
    <addaction name="menuExport"/>
    <addaction name="actionSaveAsImage"/>
    <addaction name="separator"/>
    <addaction name="actionPluginSettings"/>
    <addaction name="separator"/>
    <addaction name="actionClose_Live_Exporter"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuScene"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidgetLayers">
   <property name="windowTitle">
    <string>Layers</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="Q3DTreeView" name="treeView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionExportToWeb"/>
   <addaction name="separator"/>
   <addaction name="actionResetCameraPosition"/>
   <addaction name="actionReload"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetConsole">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Console</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QListWidget" name="listWidgetDebugView"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&gt;&gt;&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditInputBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionSTL_format">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export as STL format...</string>
   </property>
  </action>
  <action name="actionWorld_Settings">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>World Settings...</string>
   </property>
  </action>
  <action name="actionPerspective">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Perspective</string>
   </property>
  </action>
  <action name="actionOrthogonal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Orthographic</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionAlwaysOnTop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Always on Top</string>
   </property>
  </action>
  <action name="actionExportToWeb">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Export to Web...</string>
   </property>
  </action>
  <action name="actionSaveAsImage">
   <property name="text">
    <string>Save Scene as Image...</string>
   </property>
  </action>
  <action name="actionResetCameraPosition">
   <property name="text">
    <string>Reset Camera Position</string>
   </property>
   <property name="shortcut">
    <string>Shift+R</string>
   </property>
  </action>
  <action name="actionOrbit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Orbit</string>
   </property>
  </action>
  <action name="actionLayer_Panel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Layer Panel</string>
   </property>
  </action>
  <action name="actionClose_Live_Exporter">
   <property name="text">
    <string>Close Live Exporter</string>
   </property>
  </action>
  <action name="actionPluginSettings">
   <property name="text">
    <string>Plugin Settings..</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Q3DView</class>
   <extends>QWidget</extends>
   <header>Qgis2threejs.q3dview</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3DTreeView</class>
   <extends>QTreeView</extends>
   <header>Qgis2threejs.q3dtreeview</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionClose_Live_Exporter</sender>
   <signal>triggered()</signal>
   <receiver>Q3DWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
